aries-site/src/examples/components/skeleton/DashBoardSkeletonExample.js

```diff
@@ -0,0 +1,99 @@
+import React, { useContext, useEffect, useState } from 'react';
+import {
+  Box,
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  CardFooter,
+  Grid,
+  Heading,
+  Page,
+  PageContent,
+  Paragraph,
+  ThemeContext,
+  ResponsiveContext,
+} from 'grommet';
+import { adjustPad } from '../../templates/Card/utils';
+import { activities } from '../card/data';
+
+const skeleton = { message: { start: 'Loading', end: 'Content Loaded' } };
+
+const columns = {
+  xsmall: ['auto'],
+  small: ['auto'],
+  medium: ['auto'],
+  large: ['auto', 'auto'],
+  xlarge: ['auto', 'auto', 'auto'],
+};
+
+export const DashBoardSkeletonExample = () => {
+  const theme = useContext(ThemeContext);
+  const breakpoint = useContext(ResponsiveContext);
+  const [loading, setLoading] = useState(true);
+  // needed for skeleton paragraph
+  const skeletonAlign = loading ? 'none' : 'start';
+  useEffect(() => {
+    setTimeout(() => setLoading(!loading), 3000);
+  }, [loading]);
+
+  return (
+    // passing skeleton to Page will skeletize all of the children that are
+    // text, button, box, paragraph, heading
+    <Page skeleton={loading ? skeleton : undefined} height="large">
+      <PageContent>
+        <Grid columns={columns[breakpoint]} gap="medium">
+          {activities.map((activity, index) => (
```

Comment:
I still think this needs to do something different during the 'loading' phase. It shouldn't be just iterating over the same activities. That would never happen in a real situation.

I'd do something like:
```jsx
const placeholders = [{title: 'placeholder' }, {title: 'placeholder'}, {title: 'placeholder'}];
const items = loading ? placeholders : activities;
```
then here on 46 so `items.map` rather than `activities.map`. You'll have to adjust the code to handle minimal content in the activity or put enough stuff in the placholder.